src/Core/Binding/IActiveSolutionBoundTracker.cs

```diff
@@ -38,6 +38,8 @@ public interface IActiveSolutionBoundTracker
         /// <summary>
         /// Event to notify subscribers when the binding status of a solution have changed.
         /// This occurs when a new solution is opened, or the SonarQube binding status of the solution changes.
+        /// The event is raised after the <see cref="ISonarQubeService"/> connection has finished
```

Comment:
Miscellaneous: added documentation to the interface and class to make it clear that the connection has changed by the time the event is raised.